app/jinja_filters.py

```diff
@@ -253,17 +253,27 @@ def should_wrap_with_fieldset_processor() -> dict[str, Callable]:
     return {"should_wrap_with_fieldset": should_wrap_with_fieldset}
 
 
+def get_min_max_values(
+    min_max: Literal["minimum", "maximum"], answer: AnswerType, default_value: int
+) -> int:
+    if answer.get(min_max, {}) and isinstance(answer[min_max]["value"], dict):
+        schema = g.get("schema")
+        if (
+            identifier := answer[min_max]["value"].get("identifier")
+        ) in schema.min_and_max_map:
```

Comment:
When would the identifier not be in `schema.min_and_max_map`?. You need to instead check it is an answer source.

Reply:
+1

Reply:
Still outstanding.